### PR TITLE
feat: Integrate MLS with AVS library

### PR DESCRIPTION
**Pull request by thisisamir98**
Integrate MLS with AVS library

### CODE REVIEW

File: src/script/calling/enum/CallMessageType.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 export enum CALL_MESSAGE_TYPE {
   CANCEL = 'CANCEL',
-  CONF_START = 'CONFSTART',
+  CONF_START = 'CONFEND',
+  CONF_END = 'CONFSTART',
```
Comment (V-Gira) [inline src/script/calling/enum/CallMessageType.ts:23]: Looks like a deliberate change, just want to make sure it's not something that got mixed up

Comment (tlebon) [reply src/script/calling/enum/CallMessageType.ts:23]: hm this is strange. id guess typo

File: src/script/calling/CallingRepository.test.ts [hunk 2]
```diff
@@ -178,7 +201,7 @@ describe('CallingRepository', () => {
       const call = new Call(
         {domain: '', id: ''},
         createConversationId(),
-        undefined,
+        CONV_TYPE.CONFERENCE,
```
Comment (tlebon) [inline src/script/calling/CallingRepository.test.ts:204]: just curious why all of these are changing to conference? any insight from your side

File: test/helper/TestFactory.js [hunk 3]
```diff
@@ -333,6 +333,8 @@ export class TestFactory {
   async exposeCallingActors() {
     await this.exposeConversationActors();
     this.calling_repository = new CallingRepository(
+      // @ts-ignore
```
Comment (tlebon) [inline test/helper/TestFactory.js:336]: why is this necessary?

File: src/script/calling/CallingRepository.test.ts (hunk 2 above)
Comment (thisisamir98) [reply src/script/calling/CallingRepository.test.ts:204]: fixes strict mode errors

File: src/script/calling/enum/CallMessageType.ts (hunk 1 above)
Comment (thisisamir98) [reply src/script/calling/enum/CallMessageType.ts:23]: yes, a typo, thanks for catching, fixed in https://github.com/wireapp/wire-webapp/pull/13449/commits/1d2e697c422bc6d209e1cc50d765821556eab373

File: test/helper/TestFactory.js (hunk 3 above)
Comment (thisisamir98) [reply test/helper/TestFactory.js:336]: strict mode error, already was there because of this.message_repository, now that the order is changed betterer thinks there is a new error added but it is an already existing old one
